src/style.css

```diff
@@ -1,3 +1,69 @@
+:root {
+  --primary-color: #00acdc;
+  --font-family: font-family: Roboto, Arial, sans-serif;
```

Comment:
There's a mistake in declaring the `--font-family` variable. You've written `font-family:` twice. It should be `--font-family: Roboto, Arial, sans-serif;`